vulture_os/toolkit/api_parser/imperva/imperva.py

```diff
@@ -205,8 +205,8 @@ def execute(self):
                         self.write_to_file(data)
                         self.imperva_last_log_file = next_log_file
                     except Exception as e:
```

Comment:
Please add a new "except requests.exceptions.HTTPError" and handle the case "e.response.status_code == 404"